ktor-utils/common/test/io/ktor/util/PipelineContractsTest.kt

```diff
@@ -166,6 +170,11 @@ class PipelineContractsTest {
 
     @Test
     fun executePipelineWithSuspension() {
+        /**
+         * Continuation doesn't support freeze in native.
+         */
+        if (PlatformUtils.IS_NATIVE) return
```

Comment:
Maybe we should explicitly skip a test if the platform is native

Reply:
How should we do it? Kotlin test(and multiplatform) doesn't support the test exclusion for the platform.

Reply:
is there in Kotlin test something similar to https://stackoverflow.com/questions/1689242/conditionally-ignoring-tests-in-junit-4?

Reply:
We have no JUnit in common, only the Kotlin Test, and as I remember it doesn't support test ignore.

Reply:
Here is all we have in common: https://kotlinlang.org/api/latest/kotlin.test/kotlin.test/ 
So annotations only, andI'm not sure if they are working in kotlin-native